test_reporting/sai_swss_invocations.py

```diff
@@ -0,0 +1,267 @@
+import argparse
+from curses.ascii import isupper
+import json
+import os
+
+from os import listdir, name
+from os.path import isfile, join
+import shutil
+
+from report_data_storage import KustoConnector
+import yaml
+'''
+for more detail, you can see
+https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/373699/sairedis-log-scanner
+'''
+parser = argparse.ArgumentParser()
```

Comment:
please add those links for getting the parameters into a method that should be invoked when start running this python script.
You can reference report_uploader.py